<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Реєстрація</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 69, 71);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(127, 180, 175);
border-radius: 20px;</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Sitka</family>
               <pointsize>26</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(32, 69, 71);</string>
             </property>
             <property name="text">
              <string>Реєстрація</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="errorLabel">
             <property name="styleSheet">
              <string notr="true">color: rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Ім'я</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="maxLength">
              <number>25</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Прізвище</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="surnameEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>25</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>По батькові</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="patronymicEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="maxLength">
              <number>255</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phoneEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="maxLength">
              <number>13</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="maxLength">
              <number>25</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Normal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <property name="rowWrapPolicy">
              <enum>QFormLayout::DontWrapRows</enum>
             </property>
             <property name="labelAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="horizontalSpacing">
              <number>70</number>
             </property>
             <property name="verticalSpacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>133</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QRadioButton" name="studentButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Я Студент</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="teacherButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Я Викладач</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <property name="labelAlignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="formAlignment">
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="rightMargin">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="nextButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="text">
              <string>Далі</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="returnButton">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">border-style: solid;
border-width: 2px;
border-color: rgb(69, 119, 108);
background-color: rgba(69, 119, 108, 125);
border-radius: 10px;</string>
             </property>
             <property name="text">
              <string>Назад</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
